src/server/req_signal.c

```diff
@@ -497,6 +497,14 @@ post_signal_req(struct work_task *pwt)
 			pjob->ji_wattr[(int) JOB_ATR_resc_released_list].at_flags &= ~ATR_VFLAG_SET;
 
 			svr_setjobstate(pjob, JOB_STATE_RUNNING, JOB_SUBSTATE_RUNNING);
+
+			if (!(pjob->ji_qs.ji_svrflags & JOB_SVFLG_SubJob)) {
+				job_attr_def[(int) JOB_ATR_Comment].at_decode(
+						&pjob->ji_wattr[(int) JOB_ATR_Comment], (char *) 0,
+						(char *) 0,
+						form_attr_comment("Job run at %s",
```

Comment:
You can combine this line with the line above and it'll be about the same length as the line above that.  The more lines you split over makes it harder to read.